src/app/screens/TodoList/components/TodoInput/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Field, reduxForm, reset } from 'redux-form';
+import { TextInput } from 'react-native';
+
+import { actionCreators } from '../../../../../redux/toDos/actions';
+
+import styles from './styles';
+
+const FORM_NAME = 'todoInput';
```

Comment:
move all form names to a file in same root as index.js of your Screen, and call it fieldFormName, where you'll put all form string constants like field name and form name